### DISCUSSION TITLE
Issue with sleap.load_model()

### DISCUSSION THREAD

**Discussion by myee827** [talmolab/sleap#1443] [Help!]
Hi, I'm trying to use the following code to use sleap.load_model() to open a model folder for single animal tracking. However, I keep getting an error where it says FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpfz0t44gh/training_config.json', even though the folder contains a training_config.json file. Here is the code I am using: 
`import gdown
import sleap
download_link = "https://drive.google.com/uc?id=1HJ8vSzoGsd5UJplYY9rUUzSiE8vYMgLs"

downloaded_filename = "230725_160006.single_instance.n=241.zip"

gdown.download(download_link, output=downloaded_filename, quiet=False)

predictor = sleap.load_model(downloaded_filename)`

Any help on this issue would be much appreciated!

**Comment by davidasamy** [comment]
Hi, the issue is that the file needs to be **unzipped** before it can be loaded in as a model. 
Try the following script and let me know:

```py
import gdown
import sleap
from zipfile import ZipFile

download_link = "https://drive.google.com/uc?id=1HJ8vSzoGsd5UJplYY9rUUzSiE8vYMgLs"

downloaded_filename = "230725_160006.single_instance.n=241.zip"

gdown.download(download_link, output=downloaded_filename, quiet=False)

with ZipFile(downloaded_filename, 'r') as zObject: # Extract file using zipfile
	zObject.extractall("extracted")

predictor = sleap.load_model(f'extracted/{downloaded_filename[:-4]}') # Load model without .zip extension to filename
```

**Comment by roomrys** [comment]
I've opened a PR for this issue: https://github.com/talmolab/sleap/pull/1445

If you unzip the file beforehand as suggested by @davidasamy, then things should run smoothly. However, `load_model` should handle the unzipping (without issue after #1445 is merged).
